Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.5
+FROM golang:1.8
```

Comment:
Mind updating the travis config if we're going to do this? (which I'm all for, btw - been meaning to do this)